python-client/giskard/ml_worker/websocket/__init__.py

```diff
@@ -351,20 +351,20 @@ class PushDetails(BaseModel):
 
 class Push(BaseModel):
     kind: PushKind
-    key: Optional[str] = None
-    value: Optional[str] = None
+    key: Optional[str]
+    value: Optional[str]
     push_title: str
     push_details: List[PushDetails]
 
 
 class PushAction(BaseModel):
     object_uuid: str
-    arguments: Optional[List[FuncArgument]] = None
+    arguments: Optional[List[FuncArgument]]
 
 
 class GetPushResponse(BaseModel):
-    contribution: Optional[Push] = None
```

Comment:
Why are you removing None by default ?

Reply:
My fault, I think I did not sync well with main